config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -3723,6 +3723,15 @@ spec:
                   the cluster only consists of one node. For more information, see
                   https://www.rabbitmq.com/rabbitmq-queues.8.html#rebalance
                 type: boolean
+              terminationGracePeriodSeconds:
+                default: 604800
+                description: 'TerminationGracePeriodSeconds is the timeout that each
+                  rabbitmqcluster pod will have to terminate gracefully It defaults
+                  to 604800 seconds ( a week long) to ensure that the container preStop
+                  lifecycle hook can finish running For more information, see: https://github.com/rabbitmq/cluster-operator/blob/main/docs/design/20200520-graceful-pod-termination.md'
```

Comment:
In the comments further up you could use dots at the end of sentences such that this description (which might get used later on for other docs) reads nicer.

Reply:
Good catch! Fixed :)